delayed_job.gemspec

```diff
@@ -13,3 +14,4 @@ Gem::Specification.new do |spec|
   spec.test_files     = Dir.glob('spec/**/*')
   spec.version        = '4.0.6'
 end
+# rubocop:enable SpaceAroundOperators
```

Comment:
Rubocop Readme https://github.com/bbatsov/rubocop has them being re-enabled, except for one-liners.